packages/webapp/src/containers/bulkSensorUploadSlice.js

```diff
@@ -24,10 +24,18 @@ const bulkSensorsUploadSlice = createSlice({
       state.loading = false;
       state.isBulkUploadSuccessful = false;
     },
+    bulkSensorsUploadReInit: (state, action) => {
+      state.loading = false;
+      state.isBulkUploadSuccessful = false;
```

Comment:
In my head:
loading = T, success = F => bulk upload has started but is still loading
loading = F, success = T => bulk upload has finished loading, and upload was success
loading = F, success = F => dormant state, means no upload is currently in process

If anything the error state may need some work, but since are currently just showing an error snack bar this should be okay.